src/gen-objects.ts

```diff
@@ -443,6 +443,7 @@ export function addImageDefinition(target: PresSlide, opt: ImageProps) {
 		flipH: opt.flipH || false,
 		transparency: opt.transparency || 0,
 		objectName: objectName,
+		shadow: opt.shadow || null
```

Comment:
There's a `correctShadowOptions` function which you can use here. It'll validate the shadow options if present, otherwise return undefined.